enable/tools/drag_tool.py

```diff
@@ -24,11 +27,19 @@ class DragTool(BaseTool):
     # The mouse button used for this drag operation.
     drag_button = Enum("left", "right")
 
-    # End the drag operation if the mouse leaves the associated component?
+    # Deprecated; on_drag_leave is the new, more flexible / intuitive means for
+    # providing this functionality
+    # Cancel the drag operation if the mouse leaves the associated component?
     # NOTE: This behavior depends on "mouse_leave" events, which in general
     # are not fired when `capture_mouse` is True (default).
     end_drag_on_leave = Bool(False)
 
+    # Do nothing, cancel or end the drag operation if the mouse leaves the
+    # associated component?
+    # NOTE: This behavior depends on "mouse_leave" events, which in general
+    # are not fired when `capture_mouse` is True (default).
```

Comment:
maybe I should make it more clear here that `on_drag_leave = 'cancel'` is equivalent to `end_drag_on_leave = True` not `on_drag_leave = 'end'`?  That confusion is sort of the root problem at hand